src/Centipede/Crawler.php

```diff
@@ -22,34 +22,33 @@ class Crawler
     private $checker;
     private $extractor;
 
+    private $urls = array();
```

Comment:
You can use short array syntax notation because https://github.com/umpirsky/centipede-crawler/blob/master/composer.json#L13